### DISCUSSION TITLE
M1 release of Platform

### DISCUSSION THREAD

**Discussion by sdelamo** [micronaut-projects/micronaut-platform#207] [General]
@melix I think we are ready for an M1 release of `micronaut-platform`. Please trigger it, when you have cycles.

**Comment by melix** [comment]
I _think_ this will be a problem, since we're going to release a `parent` which contains a SNAPSHOT version: https://github.com/micronaut-projects/micronaut-platform/blob/master/gradle/libs.versions.toml#L87

**Reply by alvarosanchez** [reply]
I can release a milestone of the Maven plugin
